src/studio-header/StudioHeader.scss

```diff
@@ -1,16 +1,13 @@
-$spacer: 1rem;
-$white: #FFFFFF;
```

Comment:
While this file *should* be getting these values from `index.scss`, and absolutely does as it is used in this component, there's still an off chance someone is importing this file somewhere where those values have been overridden. I think just *not* removing these is likely the safest option for this PR.